Applications/Viqui/vqTreeView.cxx

```diff
@@ -1030,6 +1030,24 @@ QList<vtkVgVideoNode*> vqTreeView::GetSelectedNodes()
   return nodeList;
 }
 
+//-----------------------------------------------------------------------------
+QList<vtkVgVideoNode*> vqTreeView::GetStarredNodes()
+{
+  QList<vtkVgVideoNode*> nodeList;
+
+  for (int i = 0; i < this->topLevelItemCount(); i++)
```

Comment:
Arf, `++i` please :wink:. Or, better, use `foreach_child(auto* item, this->invisibleRootItem())`.